protocol-definitions/AtomicLong.yaml

```diff
@@ -39,103 +46,68 @@ methods:
     request:
       retryable: false
       acquiresResource: false
-      partitionIdentifier: name
+      partitionIdentifier: -1
       params:
-        - name: name
-          type: String
+        - name: groupId
+          type: RaftGroupId
           nullable: false
           since: 2.0
           doc: |
-            The name of this IAtomicLong instance.
-        - name: function
-          type: Data
-          nullable: false
-          since: 2.0
-          doc: |
-            The function applied to the currently stored value.
-    response: {}
-  - id: 3
-    name: alterAndGet
-    since: 2.0
-    doc: |
-      Alters the currently stored value by applying a function on it and gets the result.
-    request:
-      retryable: false
-      acquiresResource: false
-      partitionIdentifier: name
-      params:
+            CP group id of this IAtomicLong instance.
         - name: name
           type: String
           nullable: false
           since: 2.0
           doc: |
-            The name of this IAtomicLong instance.
+            Name of this IAtomicLong instance.
         - name: function
           type: Data
           nullable: false
           since: 2.0
           doc: |
             The function applied to the currently stored value.
-    response:
-      params:
-        - name: response
-          type: long
+        - name: returnValueType
+          type: int
           nullable: false
           since: 2.0
           doc: |
-            The result of the function application.
-  - id: 4
-    name: getAndAlter
-    since: 2.0
-    doc: |
-      Alters the currently stored value by applying a function on it on and gets the old value.
-    request:
-      retryable: false
-      acquiresResource: false
-      partitionIdentifier: name
-      params:
-        - name: name
-          type: String
-          nullable: false
-          since: 2.0
-          doc: |
-            The name of this IAtomicLong instance.
-        - name: function
-          type: Data
-          nullable: false
-          since: 2.0
-          doc: |
-            The function applied to the currently stored value.
+            0 returns the old value, 1 returns the new value
     response:
       params:
         - name: response
           type: long
           nullable: false
           since: 2.0
           doc: |
-            The old value before the function application.
-  - id: 5
+            TODO DOC
```

Comment:
TODO?

Reply:
I didn't add that. I just renamed `CPAtomicLong` to `AtomicLong`. And I see there are 290 `TODO DOC`s in the protocol repo. 🤷‍♂